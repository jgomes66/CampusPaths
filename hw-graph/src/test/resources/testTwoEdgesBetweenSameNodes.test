# Test Two Edges Between Same Nodes
# Testing to see if we can add two edges with different labels to the same two nodes

# Create a graph
CreateGraph newGraph

# Add two nodes to graph
AddNode newGraph node1
AddNode newGraph node2

# Add two edges with node1 as parent and node2 as child for both
AddEdge newGraph node1 node2 e1
AddEdge newGraph node1 node2 e2

# Print the outgoing edges from node1
ListChildren node1 from newGraph