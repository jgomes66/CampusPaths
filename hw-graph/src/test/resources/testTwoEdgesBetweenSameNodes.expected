# Test Two Edges Between Same Nodes
# Testing to see if we can add two edges with different labels to the same two nodes

# Create a graph
created graph newGraph

# Add two nodes to graph
Added node node1 to newGraph
Added node node2 to newGraph

# Add two edges with node1 as parent and node2 as child for both
Added edge e1 from node1 to node2 in newGraph
Added edge e2 from node1 to node2 in newGraph

# Print the outgoing edges from node1
The children of node1 in newGraph are: node2(e1) node2(e2)